common/utils.js

```diff
@@ -795,6 +795,24 @@ function validateDestinationChain(chains, destinationChain) {
     validateChain(chains, destinationChain);
 }
 
+async function calculateItsCrossChainGas(options) {
```

Comment:
```suggestion
async function estimateITSFee(options) {
```
reuse the same name as API for clarity